graph/resolver/discussion.resolvers.go

```diff
@@ -203,7 +202,7 @@ func (r *discussionResolver) UpcomingContent(ctx context.Context, obj *model.Dis
 }
 
 func (r *discussionResolver) FlairTemplates(ctx context.Context, obj *model.Discussion) ([]*model.FlairTemplate, error) {
```

Comment:
Can you add comments with something easiliy searchable? Like `// @deprecated` just so it's easier to come clean up in the future?